### ISSUE TITLE
reduction.percent doesn't work

### ISSUE THREAD

**Issue by akropivko** [aimeos/ai-admin-jqadm#183]
**Environment**
1. Version 2020.10.6
2. Integration  - Laravel
3. System - Windows

I've added Reduction Decorator to PrePay provider and set 3% reduction, see the screenshot below 
![Screenshot_44](https://user-images.githubusercontent.com/66216139/111079584-49885680-8503-11eb-9ba6-4bce9f777436.jpg)

But no reduction in the basket - screenshot
![Screenshot_45](https://user-images.githubusercontent.com/66216139/111079649-82283000-8503-11eb-94ab-38293ade57d0.jpg)

Probably this is because I can't input the reduction value in the same format as it is in manual?  - 
`reduction.percent : {"EUR":3}`

But the admin backend doesn't allow me to input something except number - screenshot
![Screenshot_46](https://user-images.githubusercontent.com/66216139/111079723-e6e38a80-8503-11eb-99f8-53936eb2ab55.jpg)

Could you please check?

**Comment by aimeos** [comment]
The description of the reduction service decorator was misleading because the decorator only reduces the delivery or payment costs and not the configured percentage on the basket total value. Furthermore, the example was wrong and the percentage is a number only. We fixed both in the documentation:

https://aimeos.org/docs/latest/manual/service-decorators/#reduction

Furthermore, in dev-master we've decided to implement the reduction service decorator as you have expected it to use the basket total and add a rebate based on the whole value but we can't backport that to 2020.10 LTS because it's a breaking change.
